cpp/ql/test/library-tests/dataflow/fields/aliasing.cpp

```diff
@@ -109,5 +109,5 @@ void taint_a_ptr(int* pa) {
 void test_field_conflation_array_content() {
   S s;
   taint_a_ptr(&s.m1);
-  sink(s.m2); //$f+:ir
+  sink(s.m2);
```

Comment:
Do we still get the desired (and none of the undesired) flow for deeper field nestings? Like `taint_a_ptr(&a.x.y);` or `a.p->y`?

Reply:
I think we do. I'll add some more tests to confirm this.

Reply:
Done in 49dd5763526089a3434ec8999becb51588514b7f. It looks we get the right flows.